checked_message_test.go

```diff
@@ -47,17 +47,18 @@ func TestJSONLoggerCheck(t *testing.T) {
 	})
 }
 
-func TestCheckedMessageIsSingleUse(t *testing.T) {
-	expected := []string{
-		`{"level":"info","msg":"single-use"}`,
-		`{"level":"error","msg":"Shouldn't re-use a CheckedMessage.","original":"single-use"}`,
-	}
-	withJSONLogger(t, nil, func(logger Logger, buf *testBuffer) {
-		cm := logger.Check(InfoLevel, "single-use")
-		cm.Write() // ok
-		cm.Write() // first re-use logs error
-		cm.Write() // second re-use is silently ignored
-		assert.Equal(t, expected, buf.Lines(), "Expected re-using a CheckedMessage to log an error.")
+func TestCheckedMessageUnsafeWrite(t *testing.T) {
+	withJSONLogger(t, opts(InfoLevel), func(logger Logger, buf *testBuffer) {
+		stub := stubExit()
+		defer stub.Unstub()
+		cm := logger.Check(InfoLevel, "bob lob law blog")
```

Comment:
it's bob loblaw's law blog!
http://arresteddevelopment.wikia.com/wiki/Bob_Loblaw